tasty-plutus/src/Test/Tasty/Plutus/WithScript.hs

```diff
@@ -58,28 +48,29 @@ import Prelude
 
  = Important note
 
- Unless your 'Validator' has been prepared using 'toTestValidator', this will
- likely not behave as intended.
+ Unless your 'TestValidator' has been prepared using 'toTestValidator', this will
+ likely not behave as intended. We use 'WrappedValidator' to prevent
+ other wrapper functions from being used.
 
  @since 3.0
```

Comment:
Due to the API change on this function, the `since` should be updated to match the version in which it changed.